doc/glossary.rst

```diff
@@ -1357,6 +1357,8 @@ functions or non-estimator constructors.
         equal weight by giving each sample a weight inversely related
         to its class's prevalence in the training data:
         ``n_samples / (n_classes * np.bincount(y))``.
+        **Note** however that this rebalancing does not take the weight of
+        samples in each class into account.
```

Comment:
Perhaps we should have a weight-loss card option. It would be interesting to see if that improved imbalanced boosting

Reply:
Apparently my phone wrote "weight-loss card" (!) there. Amended.